hack/bump-chart.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+export file=chart/btp-manager/Chart.yaml
```

Comment:
Chart.yaml is generated by `gen-chart.sh` from make (target `module-operator-chart`)